src/node/esbuildService.ts

```diff
@@ -102,7 +102,7 @@ export const transform = async (
     if (e.errors) {
       e.errors.forEach((m: Message) => printMessage(m, src))
     } else {
-      console.error(e)
+      throw new Error(e);
```

Comment:
This will broke the runing app.

Reply:
I see. I think we should add error handling in cli.js, What do you think?

Reply:
I think `console.error` is  ok.

Reply:
OK. Well, sorry for the lack of explanation. I have to explain why I created this pull request.

I'm using vite to build my React app. When I attempted to build my app on Github Actions, vite occurred some error, but an action workflow didn't stop properly. I expect actions are stopped and print an error if some errors occur, but tests seem to succeed. This is a problem, I think.

Reply:
I think `throw new Error(e)` is better than `console.error(e)`. Because `throw new Error` can print error description with source map.

Reply:
> When I attempted to build my app on Github Actions, vite occurred some error, but an action workflow didn't stop properly. I expect actions are stopped and print an error if some errors occur, but tests seem to succeed. This is a problem. I think vite has to stop with an error

The `console.error` can print error, can you give a reporduction?

Reply:
https://github.com/R-NK/vite-warning-bugs/runs/1001561854?check_suite_focus=true
I've created a reproduction to explain this case.
Please see the `build` section. `Range Error` occurred while transforming, but the process weren't killed. Even though bundled js is empty, CI says all green.

Reply:
Yeah.I got it.

Reply:
I would prefer 
```ts
console.error(e)
process.exit(1)
```